tests/performance/jshint.js

```diff
@@ -9596,10 +9596,10 @@ assert.ok = ok;
 
 // 5. The equality assertion tests shallow, coercive equality with
 // ==.
-// assert.equal(actual, expected, message_opt);
+// assert.strictEqual(actual, expected, message_opt);
 
-assert.equal = function equal(actual, expected, message) {
-  if (actual != expected) fail(actual, expected, message, '==', assert.equal);
+assert.strictEqual = function equal(actual, expected, message) {
```

Comment:
Would it make more sense to reconfigure no-restricted-properties for this file to allow the local assert.equal? I'm assuming this is third party code, maybe I'm misunderstanding.

Reply:
Oops, this was an accidental change resulting from a global find/replace.